2022-08-29 07:20:24,011 INFO    Thread-3096:116206 [wandb_setup.py:_flush():75] Loading settings from /home/mgatlin3/.config/wandb/settings
2022-08-29 07:20:24,012 INFO    Thread-3096:116206 [wandb_setup.py:_flush():75] Loading settings from wandb/settings
2022-08-29 07:20:24,012 INFO    Thread-3096:116206 [wandb_setup.py:_flush():75] Loading settings from environment variables: {'project': 'sweep-FT', 'entity': 'mgatlin3', 'root_dir': '/home/mgatlin3/FT', 'run_id': 'hszsznc6', 'sweep_param_path': '/home/mgatlin3/FT/wandb/sweep-paq5irxm/config-hszsznc6.yaml', 'sweep_id': 'paq5irxm'}
2022-08-29 07:20:24,012 INFO    Thread-3096:116206 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'sweep.py', 'program': '/home/mgatlin3/FT/sweep.py'}
2022-08-29 07:20:24,012 INFO    Thread-3096:116206 [wandb_init.py:_log_setup():437] Logging user logs to /home/mgatlin3/FT/wandb/run-20220829_072024-hszsznc6/logs/debug.log
2022-08-29 07:20:24,012 INFO    Thread-3096:116206 [wandb_init.py:_log_setup():438] Logging internal logs to /home/mgatlin3/FT/wandb/run-20220829_072024-hszsznc6/logs/debug-internal.log
2022-08-29 07:20:24,012 INFO    Thread-3096:116206 [wandb_init.py:init():471] calling init triggers
2022-08-29 07:20:24,012 INFO    Thread-3096:116206 [wandb_init.py:init():474] wandb.init called with sweep_config: {'batch_size': 16, 'dropout_c': 0, 'dropout_ga': 0.4, 'epochs': 60, 'learning_rate': 0.01, 'num_features_c1': 16, 'num_features_c2': 64, 'num_features_c3': 128, 'num_features_c4': 32}
config: {'batch_size': 4, 'epochs': 5, 'dropout_c': 0.3, 'dropout_ga': 0.1, 'num_features_c1': 64, 'num_features_c2': 64, 'num_features_c3': 128, 'num_features_c4': 64, 'learning_rate': 0.0003}
2022-08-29 07:20:24,012 INFO    Thread-3096:116206 [wandb_init.py:init():524] starting backend
2022-08-29 07:20:24,012 INFO    Thread-3096:116206 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-08-29 07:20:24,016 INFO    Thread-3096:116206 [backend.py:ensure_launched():217] starting backend process...
2022-08-29 07:20:24,198 INFO    Thread-3096:116206 [backend.py:ensure_launched():222] started backend process with pid: 8063
2022-08-29 07:20:24,199 INFO    Thread-3096:116206 [wandb_init.py:init():533] backend started and connected
2022-08-29 07:20:24,207 INFO    Thread-3096:116206 [wandb_run.py:_config_callback():1131] config_cb None None {'batch_size': 16, 'dropout_c': 0, 'dropout_ga': 0.4, 'epochs': 60, 'learning_rate': 0.01, 'num_features_c1': 16, 'num_features_c2': 64, 'num_features_c3': 128, 'num_features_c4': 32}
2022-08-29 07:20:24,211 INFO    Thread-3096:116206 [wandb_init.py:init():597] updated telemetry
2022-08-29 07:20:24,571 INFO    Thread-3096:116206 [wandb_init.py:init():628] communicating run to backend with 30 second timeout
2022-08-29 07:20:25,213 INFO    Thread-3096:116206 [wandb_run.py:_on_init():1923] communicating current version
2022-08-29 07:20:25,304 INFO    Thread-3096:116206 [wandb_run.py:_on_init():1927] got version response upgrade_message: "wandb version 0.13.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-08-29 07:20:25,304 INFO    Thread-3096:116206 [wandb_init.py:init():659] starting run threads in backend
2022-08-29 07:20:29,593 INFO    Thread-3096:116206 [wandb_run.py:_console_start():1897] atexit reg
2022-08-29 07:20:29,594 INFO    Thread-3096:116206 [wandb_run.py:_redirect():1770] redirect: SettingsConsole.REDIRECT
2022-08-29 07:20:29,594 INFO    Thread-3096:116206 [wandb_run.py:_redirect():1775] Redirecting console.
2022-08-29 07:20:29,594 ERROR   Thread-3096:116206 [wandb_run.py:_redirect():1834] Failed to redirect.
Traceback (most recent call last):
  File "/home/mgatlin3/.local/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 1827, in _redirect
    out_redir.install()
  File "/home/mgatlin3/.local/lib/python3.9/site-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/home/mgatlin3/.local/lib/python3.9/site-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/home/mgatlin3/.local/lib/python3.9/site-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/home/mgatlin3/.conda/envs/poop/lib/python3.9/signal.py", line 56, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread of the main interpreter
2022-08-29 07:20:29,595 INFO    Thread-3096:116206 [wandb_init.py:init():684] run started, returning control to user process
2022-08-29 07:20:29,595 INFO    Thread-3096:116206 [wandb_config.py:__setitem__():150] config set architecture_name = CNN - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7ffad0c1d3a0>>
2022-08-29 07:20:29,595 INFO    Thread-3096:116206 [wandb_run.py:_config_callback():1131] config_cb architecture_name CNN None
2022-08-29 09:35:46,823 INFO    Thread-3096:116206 [wandb_run.py:_finish():1685] finishing run mgatlin3/sweep-FT/hszsznc6
2022-08-29 09:35:46,823 INFO    Thread-3096:116206 [wandb_run.py:_atexit_cleanup():1866] got exitcode: 0
2022-08-29 09:35:46,827 INFO    Thread-3096:116206 [wandb_run.py:_restore():1838] restore
2022-08-29 09:35:46,830 INFO    Thread-3096:116206 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 1
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1510289
  total_bytes: 1510289
}

2022-08-29 09:35:46,962 INFO    Thread-3096:116206 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 1
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1510289
  total_bytes: 1510289
}

2022-08-29 09:35:47,840 INFO    Thread-3096:116206 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 4
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1510289
  total_bytes: 1513571
}

2022-08-29 09:35:47,942 INFO    Thread-3096:116206 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 3377
  total_bytes: 1520115
}

2022-08-29 09:35:48,043 INFO    Thread-3096:116206 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1520115
  total_bytes: 1520115
}

2022-08-29 09:35:48,144 INFO    Thread-3096:116206 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1520115
  total_bytes: 1520115
}

2022-08-29 09:35:48,246 INFO    Thread-3096:116206 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1520115
  total_bytes: 1520115
}

2022-08-29 09:35:48,348 INFO    Thread-3096:116206 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1520115
  total_bytes: 1520115
}

2022-08-29 09:35:48,450 INFO    Thread-3096:116206 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1520115
  total_bytes: 1520115
}

2022-08-29 09:35:48,551 INFO    Thread-3096:116206 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1520115
  total_bytes: 1520115
}

2022-08-29 09:35:48,653 INFO    Thread-3096:116206 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1520115
  total_bytes: 1520115
}

2022-08-29 09:35:48,847 INFO    Thread-3096:116206 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1520115
  total_bytes: 1520115
}

2022-08-29 09:35:49,054 INFO    Thread-3096:116206 [wandb_run.py:_on_finish():1995] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1520115
  total_bytes: 1520115
}
local_info {
}

2022-08-29 09:35:50,700 INFO    Thread-3096:116206 [wandb_run.py:_footer_history_summary_info():3102] rendering history
2022-08-29 09:35:50,701 INFO    Thread-3096:116206 [wandb_run.py:_footer_history_summary_info():3134] rendering summary
2022-08-29 09:35:50,705 INFO    Thread-3096:116206 [wandb_run.py:_footer_sync_info():3057] logging synced files
